app/models/lettings_log.rb

```diff
@@ -132,7 +132,7 @@ class LettingsLog < Log
           illness_type_10: false)
   }
 
-  scope :filter_by_user_text_search, ->(param, user) { where(assigned_to: user.support? ? User.search_by(param) : User.own_and_managing_org_users(user.organisation).search_by(param)) }
+  scope :filter_by_user_text_search, ->(param, user) { where(assigned_to: User.visible(user).search_by(param)) }
```

Comment:
I think this would also work:
`scope :filter_by_user_text_search, ->(param, user) { where(assigned_to: visible(user).search_by(param)) }`

Reply:
`visible` is the scope on the user model we want to determine which user's the current user has visibility of. Removing `User.` here would result in it using the `visible` scope within the Log model not the User model. I'm going to leave it unchanged.